test/platform/test-platform-fixtures.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ * Fixtures used across internal tests
+ */
+export function makeSendCommand(
```

Comment:
Functions that are exported should have some JSDoc.

Reply:
Done